RevPi-HAT-EEPROM-Format.md

```diff
@@ -1,145 +1,80 @@
-  1       pslen       product string length (bytes)
-  X       vstr        ASCII vendor string e.g. "ACME Technology Company"
-  Y       pstr        ASCII product string e.g. "Special Sensor Board"
-```
-
-##### Device Tree Attributes
+### Device Tree Attributes
 
 | Field           | ProcFS                              | Type         | Example                                |
 |-----------------|-------------------------------------|--------------|----------------------------------------|
 | UUID            | `/proc/device-tree/hat/uuid`        | ASCII string | `9362d4cc-c3d8-4de6-0000-55d08e938e60` |
-| product ID      | `/proc/device-tree/hat/product`     | ASCII string | `0x0001`                               |
+| product ID      | `/proc/device-tree/hat/product_id`  | ASCII string | `0x0001`                               |
 | product version | `/proc/device-tree/hat/product_ver` | ASCII string | `0x0078`                               |
 | vendor string   | `/proc/device-tree/hat/vendor`      | ASCII string | `KUNBUS GmbH`                          |
 | product string  | `/proc/device-tree/hat/product`     | ASCII string | `RevPi Core`                           |
 
-#### GPIO map atom data (type=0x0002):
-
-  GPIO map for bank 0 GPIO on 40W B+ header.
-
-  **NOTE** GPIO number refers to BCM2835 GPIO number and **NOT** J8 pin number!
-
-```text
-  Bytes   Field
-  1       bank_drive  bank drive strength/slew/hysteresis, BCM2835 can only set per bank, not per IO
-            Bits in byte:
-            [3:0] drive       0=leave at default, 1-8=drive*2mA, 9-15=reserved
-            [5:4] slew        0=leave at default, 1=slew rate limiting, 2=no slew limiting, 3=reserved
-            [7:6] hysteresis  0=leave at default, 1=hysteresis disabled, 2=hysteresis enabled, 3=reserved
-  1       power
-            [1:0] back_power  0=board does not back power Pi
-                              1=board back powers and can supply up to 1.3A to the Pi
-                              2=board back powers and can supply up to 2A to the Pi
-                              3=reserved
-                              If back_power=2 high current USB mode is automatically enabled.
-            [7:2] reserved    set to 0
-  28      1 byte per IO pin
-            Bits in each byte:
-            [2:0] func_sel    GPIO function as per FSEL GPIO register field in BCM2835 datasheet
-            [4:3] reserved    set to 0
-            [6:5] pulltype    0=leave at default setting,  1=pullup, 2=pulldown, 3=no pull
-            [  7] is_used     1=board uses this pin, 0=not connected and therefore not used
-```
-
-#### Device Tree atom data (type=0x0003):
+The RevPi Hat EEPROM Format uses the custom atoms of the RPi Hat Spec. As the custom atoms can’t be renamed the index of the custom atom is important and can’t be changed without a new version of this specification. The table shows the custom attributes defined for the RevPi HAT EEPROM Format:
```

Comment:
Which table shows this? The one above those line or the one below this line? Would prefer a table caption and a reference to the table, but that is not that easy to add in markdown. Give a hint, if the table above or below is meant.